### PR TITLE
Update the file pattern in Urmp base dataloader

### PR DESCRIPTION
**Pull request by lukewys**
Update the file pattern in Urmp base dataloader to match the file pattern in https://console.cloud.google.com/storage/browser/magentadata/datasets/urmp/urmp_20210324.

### CODE REVIEW

File: ddsp/training/data.py [hunk 1]
```diff
@@ -411,7 +411,11 @@ def features_dict(self):
 class Urmp(TFRecordProvider):
   """Urmp training set."""
 
-  def __init__(self, base_dir, instrument_key='tpt', split='train'):
+  def __init__(self,
+               base_dir,
+               instrument_key='tpt',
+               split='train',
+               batched='batched'):
```
Comment (jesseengel) [inline ddsp/training/data.py:418]: Maybe be a boolean batched=True, and then lookup the string from there?

File: ddsp/training/data.py [hunk 2]
```diff
@@ -420,19 +424,22 @@ def __init__(self, base_dir, instrument_key='tpt', split='train'):
         ['all', 'bn', 'cl', 'db', 'fl', 'hn', 'ob', 'sax', 'tba', 'tbn',
         'tpt', 'va', 'vc', 'vn'].
       split: Choices include ['train', 'test'].
+      batched: Choices include ['batched', 'unbatched'].
```
Comment (jesseengel) [inline ddsp/training/data.py:427]: Boolean, true, false. Describe what the difference is?

**Review by jesseengel — COMMENTED** [review]
One quick change

**Review by jesseengel — APPROVED** [review]
Okay, I'll pull these in.

File: ddsp/training/data.py [hunk 3]
```diff
@@ -420,19 +424,30 @@ def __init__(self, base_dir, instrument_key='tpt', split='train'):
...
+        With the "batched" suffix, the dataloader will load tfrecords
+        containing segmented audio samples in 4 seconds. With the "unbatched"
+        suffix, the dataloader will load tfrecords containing unsegmented
+        samples which could be used for learning note sequence in URMP dataset.
+
     """
     self.instrument_key = instrument_key
     self.split = split
     self.base_dir = base_dir
+    self.suffix = suffix if suffix is None else '_' + suffix
```
Comment (ethman) [inline ddsp/training/data.py:440]: I think `f'{None}'` prints `'None'` here, not an empty string. You probably want to change this line to:

    self.suffix = suffix if '' is None else '_' + suffix

File: ddsp/training/data.py [hunk 4]
```diff
@@ -420,19 +424,30 @@ def __init__(self, base_dir, instrument_key='tpt', split='train'):
         ['all', 'bn', 'cl', 'db', 'fl', 'hn', 'ob', 'sax', 'tba', 'tbn',
         'tpt', 'va', 'vc', 'vn'].
       split: Choices include ['train', 'test'].
+      suffix: Choices include [None, 'batched', 'unbatched'], but broadly
+        applies to any suffix adding to the file pattern.
+        When suffix is not None, will add "_suffix" to the file pattern.
```
Comment (ethman) [inline ddsp/training/data.py:429]: nit: `"_suffix"` --> `"_{suffix}"`

**Comment by ethman** [comment]
LGTM

**Pull request merged**